include/ofi_util.h

```diff
@@ -195,6 +195,7 @@ struct util_domain {
 	uint64_t		info_domain_caps;
 	uint64_t		info_domain_mode;
 	int			mr_mode;
+	size_t			addrlen;
```

Comment:
A domain can support multiple address lengths.  There could be an AV for ipv4 and a separate AV for ipv6.  Please remind me why the domain has an address format.  I remember there being a reason, but I don't recall what.

Reply:
I see. Please, take a look into the last commit. It shouldn't be an issue anymore.